src/style.css

```
@@ -1 +1,105 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+}
+
+/* #region bar */
+.search-bar {
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  text-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+  position: relative;
+  height: 70px;
+}
+
+.search-bar__small {
+  height: 42px;
+}
+
+.search-bar::after {
+  background-image: url(./images/Search.svg);
+  flex-shrink: 0;
+  background-size: contain;
+  content: "";
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+}
+
+.search-bar__small::after {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+}
+
+.search-bar:hover {
+  background: #FFF;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+.search-bar:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+}
+
+.search-bar input[type="text"] {
```

do not use such selectors (combined)
you have 1 class for it
```suggestion
.search-bar-text {
```